src/ast/helpers/stmt_create_table.rs

```diff
@@ -66,6 +66,7 @@ pub struct CreateTableBuilder {
     pub clone: Option<ObjectName>,
     pub engine: Option<String>,
     pub comment: Option<String>,
+    pub autoincrement_offset: Option<u32>,
```

Comment:
is there any reason to use a different style than the original `auto_increment`? I wonder if you considered:

```suggestion
    pub auto_increment_offset: Option<u32>,
```
?

If so can you explain why you chose this way?

I was thinking that following the exisiting convention would make the relevant code / field easier to search for (aka you can search for `auto_increment` in the code to see how `auto_increment` in `SQL` would be parsed.

Reply:
No specific reason, your suggestion of more closely following convention is a better option I think, so I updated the PR